config/manifests/kustomization.yaml

```diff
@@ -31,3 +31,7 @@ resources:
 
 patchesStrategicMerge:
 - patch-delete-user-cluster-role.yaml
+images:
+- name: docker.io/apache/camel-k
+  newName: docker.io/apache/camel-k
+  newTag: 1.11.0-SNAPSHOT
```

Comment:
Do we need to commit this?  I thought it hasn't been committed deliberately.

Reply:
I've removed them for now. However they are introduced by the `make bundle` procedure. I'll open a follow up request to discuss if that is valid or not.